### PR TITLE
Fix deploy script

### PR DESCRIPTION
**Pull request by jsf9k**

### CODE REVIEW

File: terraform/deploy_new_portscan_instance.sh [hunk 1]
```diff
@@ -27,7 +27,7 @@ terraform workspace select "$workspace"
 # Strip control characters, then look for the text "id" surrounded by
 # space characters, then extract only the ID from that line.
 nmap_instance_id=$(terraform state show aws_instance.cyhy_nmap[$index] | \
-                       sed -r "s/\x1b\[[0-9;]*m?//g" | \
+                       sed $'s,\x1b\\[[0-9;]*[[:alpha:]]],,g' | \
```
Comment (dav3r) [inline terraform/deploy_new_portscan_instance.sh]: This does NOT work correctly on OSX:
`sed $'s,\x1b\\[[0-9;]*[[:alpha:]]],,g'`, however this does:
`sed $'s,\x1b\\[[0-9;]*[[:alpha:]],,g'`

Not sure where that extra `]` came from, but it wasn't there when you gave me samples to test.

**Review by dav3r — CHANGES_REQUESTED** [review]
See my note for a slight change required to make this work with the BSD sed on OSX.

File: terraform/deploy_new_portscan_instance.sh (hunk 1 above)
Comment (jsf9k) [reply terraform/deploy_new_portscan_instance.sh]: Good eye.  I fat fingered that.  Fixed in 803e622.

**Review by dav3r — APPROVED** [review]
Looking good! 🎉

**Pull request merged**